src/engine/commands/configure/index.ts

```diff
@@ -3,16 +3,18 @@ import { Context } from '../../../common/context';
 import { translations } from '../../../common/translations';
 import { Interactive } from '../../../common/interactive';
 import { DEFAULT_ENVIRONMENT_NAME } from '../../../consts/Environment';
+import _ = require('lodash');
```

Comment:
What do you think about adding import eslint plugin? It is very useful and it can autoformat code. You can see an example here https://github.com/8base-services/app-builder-v2/blob/f548db9ea2428a4247253f57396a879746f7adf2/.eslintrc.js#L43

Reply:
Yes, why not
But, to avoid big changes we could do this changes in an another pull request.